### PR TITLE
Update reason snippets after 3.13

### PR DESCRIPTION
**Pull request by jchavarri**

### CODE REVIEW

File: docs/communicate-with-javascript.md [hunk 1]
```diff
@@ -855,14 +855,13 @@ let add = [%mel.raw {|
 let () = Js.log (add 1 2)
 ```
 ```reasonml
-let add = [%mel.raw
-  {|
+let add =
+  {%mel.raw |
```
Comment (jchavarri) [inline docs/communicate-with-javascript.md:859]: @anmonteiro This change is a bit strange, is it expected? If that's the way to write this now, we should probably adapt the references across the docs cc @feihong

Comment (feihong) [reply docs/communicate-with-javascript.md:859]: Wow, this is huge change. I see no mention of it in https://github.com/reasonml/reason/blob/master/CHANGES.md.

@jchavarri This PR, when merged, will update Playground to Reason 3.13, then? Then I guess we should also change this line from 3.12 to 3.13:

https://github.com/melange-re/melange-re.github.io/blob/master/playground/src/app.jsx#L148

**Review by feihong — APPROVED** [review]
lgtm

File: docs/communicate-with-javascript.md (hunk 1 above)
Comment (feihong) [reply docs/communicate-with-javascript.md:859]: Is `{%mel.raw ||}` supported in OCaml syntax?

**Pull request merged**